sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -245,6 +291,36 @@ func (m *networkFlowManager) enrichAndSend() {
 	}
 }
 
+func (m *networkFlowManager) enrichAndSendProcesses() {
+	currentProcesses := m.currentEnrichedProcesses()
+
+	updatedProcesses := computeUpdatedProcesses(currentProcesses, m.enrichedProcessesLastSentState)
+
+	m.enrichedProcessesLastSentState = currentProcesses
+
+	if len(updatedProcesses) == 0 {
+		return
+	}
+
+	processesToSend := &central.ProcessListeningOnPortsUpdate{
+		ProcessesListeningOnPorts: updatedProcesses,
+		Time:                      types.TimestampNow(),
+	}
+
+	metrics.IncrementTotalProcessesSentCounter(len(processesToSend.ProcessesListeningOnPorts))
+	log.Debugf("Processes update : %v", processesToSend)
```

Comment:
Do you have an example of how does this look like in sensor debug logs? Using `%v` will print all struct's values. How useful do you think that will be for someone debugging an issue here? If you think _all_ properties should be shown, perhaps use `%+v`, which also prints the property names. This is specially good if you have string values in the struct: [example](https://go.dev/play/p/VjRjn468SeW).

Reply:
Just notice you mentioned in the readme ` Get rid of some of the additional logging that has been added once we are done testing manually.`. 

I assume you were referencing this log statement?

Reply:
Yeah, I need to go back through the entire PR and tidy up logging, this being one of the messages.